### ISSUE TITLE
Add translations for CC-BY-NC-SA

### ISSUE THREAD

**Issue by santisoler** [santisoler/cc-licenses#12]
After #10 is merged, we would need to add the translated versions of the CC-BY-NC-SA license to Spanish and German.

TODO:
- [x] Add Spanish translation of CC BY-NC-SA
- [x] Add German translation of CC BY-NC-SA

**Comment by cyb3rko** [comment]
I can do this if I find some time for it.
Greetings

**Comment by santisoler** [comment]
Awesome @cyb3rko !
Let's wait until #10 is merged and then you can start. Do you agree?

**Comment by Magicianred** [comment]
Do you need Italian translation?

**Comment by santisoler** [comment]
Hi @Magicianred !

I've started this repo for personal usage, but it ended up getting a lot of attention.
@cyb3rko opened a PR for adding a German translation and I thought that would be awesome to have.

So, if you want to add an Italian translation, feel free to open a PR! I'll be glad to have that!
(Would be awesome to try to remember the little Italian I learned at school!)

I'll open a new Issue for adding an Italian translation, so we can discuss there instead of here.

Thanks for reaching out!

**Comment by weensy** [comment]
@santisoler 
It is about another language,
can I add Japanese and Korean translations this repo?
I'm a native Korean speaker and I live in Japan.

**Comment by santisoler** [comment]
Hi @vaporwavy! Sure, feel free to add them. As I mention before, this repo started just as a personal deposit for adding CC Licenses to my works, but as it caught attention, we started adding as many translations as needed by other contributors.
So feel free to add any translation you might want to use.
